lib/MySQL_Session.cpp

```diff
@@ -2851,7 +2848,6 @@ bool MySQL_Session::handler_again___status_CHANGING_AUTOCOMMIT(int *_rc) {
 		myds->DSS = STATE_MARIADB_GENERIC;
 		NEXT_IMMEDIATE_NEW(st);
 	} else {
-		MyHGM->p_update_mysql_error_counter(p_mysql_error_type::mysql, myconn->parent->myhgc->hid, myconn->parent->address, myconn->parent->port, mysql_errno(myconn->mysql));
```

Comment:
Instead of replacing this with 3 calls, we can use this:
```
MyHGM->p_update_mysql_error_counter(p_mysql_error_type::mysql, myconn->parent->myhgc->hid, myconn->parent->address, myconn->parent->port, ( mysql_errno(myconn->mysql) ? mysql_errno(myconn->mysql) : ER_PROXYSQL_OFFLINE_SRV ) );
```